### PR TITLE
Update the typedoc build

### PR DESCRIPTION
**Pull request by PanayotCankov**

### CODE REVIEW

File: tns-core-modules/color/color.d.ts [hunk 1]
```diff
@@ -1,6 +1,9 @@
 ﻿/**
+ * @module "color"
+ * 
  * Allows creating colors to be used when styling the UI.
- */
+ */ /** */
```
Comment (vakrilov) [inline tns-core-modules/color/color.d.ts]: are those double comments(` */ /** */`) on purpose

Comment (PanayotCankov) [inline tns-core-modules/color/color.d.ts]: This is a "known bug" I don't plan to fix for the typedoc-plugin-external-module-name,
https://github.com/christopherthielen/typedoc-plugin-external-module-name#using

Comment (PanayotCankov) [inline tns-core-modules/color/color.d.ts]: There is existing PR here: https://github.com/christopherthielen/typedoc-plugin-external-module-name/pull/9

**Comment by vchimev** [comment]
The PR build is hanging on this ...

**Comment by PanayotCankov** [comment]
No it is not :P
Just amended the wild RegEx-es in the grunt file it should be fine now.

**Pull request merged**